docs/howto/set-config.rst

```diff
@@ -37,6 +37,12 @@ or by creating a corresponding environment variable:
 .. code-block:: bash
 
     export AIRFLOW__CORE__SQL_ALCHEMY_CONN=my_conn_string
+    
+For setting boolean configuration values use ``true``/``false``:
+
+.. code-block:: bask
+
+    export AIRFLOW__CORE__REMOTE_LOGGING=true
```

Comment:
It shouldn't matter. the values are read as string by the configparser

Reply:
```python
❯ export AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS=True
❯ ipython
Python 3.7.4 (default, Aug 13 2019, 15:17:50)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.8.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from airflow.configuration import  conf

In [2]: conf.getboolean("core", "load_default_connections")
Out[2]: True

In [3]:
Do you really want to exit ([y]/n)? y
❯ export AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS=False
❯ ipython
Python 3.7.4 (default, Aug 13 2019, 15:17:50)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.8.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from airflow.configuration import  conf

In [2]: conf.getboolean("core", "load_default_connections")
Out[2]: False

In [3]:
```

Reply:
Check https://github.com/apache/airflow/blob/096f5c5cba963b364ee75f6686d128cd4d34d66e/airflow/configuration.py#L330

Reply:
thanks @kaxil for getting into that. When I set the environmental variables I was not aware that airflow parses booleans that gracefully and supports many variants (`1`, `TRUE`, `true`, `t`, `True`, ...). One could avoid concerns by giving an example in the documentation, like I did above.

Reply:
Even `tRuE` should work :) as we convert them to `.lower()` so it doesn't matter for a user until there is a Typo :)

If you think some docs around that can help ease the confusion, I am happy for you to update the docs in this PR and merge it :)

Reply:
Maybe I was thinking too much like a software developer here. And as the parsing is so clever, people won't have issues. 

I will close the PR, thanks for your feedback.